lua/ulx_yeet/sh_yeet.lua

```diff
@@ -172,6 +179,7 @@ local function playerDrop( ply, ent )
         if not IsValid( ent ) then return end
 
         local ragdoll = ragdollPlayer( ent, newVelocity * 50 )
```

Comment:
Not in scope of this PR, but out of curiosity: Is this something that could be made a config option? Seems magic-y to me

Reply:
This value is to compensate for setting the players velocity when let loose, for some reason by default ragdoll barely gets thrown so i'm making sure its the same as a player would be flung. Don't think this one needs to be configurable.